### PR TITLE
Choice uniqueness

### PR DESCRIPTION
**Pull request by Fanderman**
The previous pull became a mess so I opened a new fresh one.

Added 2 tests for vote view, if needs be I can create more similar tests.

Fixed the issue with two same answers being counted as separate answers.

Made a few imports in tests.

### CODE REVIEW

File: izp/polls/tests.py [hunk 1]
```diff
@@ -283,6 +283,52 @@ def setUp(self):
...
+            url, {'is_open': True,
+                  'code': password,
+                  'new_choice': 'odpowiedz'})
+        password = open_question.accesscode_set.all()[1].code
+        response = self.client.post(
+            url, {'is_open': True,
+                  'code': password,
+                  'new_choice': 'odpowiedz'})
+        count2 = 0
+        for c in Choice.objects.all():
```
Comment (czapiga) [inline izp/polls/tests.py]: You want to filter choices that are not connected with question created in test. I'm pretty sure I have added similar comment in your previous PR. That's the lost history of review I was talking about.

File: izp/polls/tests.py [hunk 2]
```diff
@@ -283,6 +283,52 @@ def setUp(self):
...
+        password = open_question.get_codes()[1]
+        response = self.client.post(
+            url, {'is_open': True,
+                  'code': password,
+                  'new_choice': 'odpowiedz'})
+        count1 = 0
+        for c in Choice.objects.all():
+            if c.votes == 1:
+                count1 = count1 + 1
+        self.assertIs(count1, 2)
```
Comment (czapiga) [inline izp/polls/tests.py]: This assert is incorrect. The same as before.

File: izp/polls/tests.py (hunk 1 above)
Comment (Fanderman) [reply izp/polls/tests.py]: Changed according to review.

File: izp/polls/tests.py [hunk 3]
```diff
@@ -283,6 +283,52 @@ def setUp(self):
...
+            url, {'is_open': True,
+                  'code': password,
+                  'new_choice': 'odpowiedz2'})
+        password = open_question.get_codes()[1]
+        response = self.client.post(
+            url, {'is_open': True,
+                  'code': password,
+                  'new_choice': 'odpowiedz'})
+        count1 = 0
+        for c in Choice.objects.all():
```
Comment (czapiga) [inline izp/polls/tests.py]: You can do better than this. HINT: ```question.choice_set```

**Comment by Fanderman** [comment]
I hope I have done it right this time.

**Pull request merged**